packages/ember-toucan-core/src/-private/assert-block-or-argument-exists.ts

```diff
@@ -0,0 +1,39 @@
+import { assert } from '@ember/debug';
+
+export type AssertBlockOrArg = {
```

Comment:
@ynotdraw  new singular function / not a helper.

Reply:
yay! This looks awesome to me!